sdk/metric/sdk.go

```diff
@@ -352,6 +359,20 @@ func (m *Accumulator) Collect(ctx context.Context) int {
 	return checkpointed
 }
 
+func (m *Accumulator) enrich(ctx context.Context, kvs []label.KeyValue) []label.KeyValue {
+	if m.enricher == nil {
+		return kvs
+	}
+	out, err := m.enricher(ctx, kvs)
+
+	// Return the enricher result if it is non-nil and no error.
+	if err == nil && out != nil {
+		return out
+	}
+	otel.Handle(err)
```

Comment:
If `out == nil`, we will invoke `otel.Handle(nil)`. I couldn't find any documentation for what `otel.Handle` is supposed to do in that case.

```suggestion
	if err != nil {
		otel.Handle(err)
		return kvs
	}
	
	if out == nil {
		return kvs
	}
	
	return out
```